src/utils/time.ts

```diff
@@ -251,3 +253,14 @@ export const getDayInterval = (date: Date | number) => ({
   start: startOfDay(date),
   end: endOfDay(date),
 });
+
+export const getDurationMinutes = function (start: string, end: string) {
+  let startDate = parseISO(start);
+  let endDate = parseISO(end);
+  return differenceInMinutes(endDate, startDate);
+};
+
+export const getUTCStartTime = function (start: string) {
+  let startDate = parseISO(start);
+  return getUnixTime(startDate);
+};
```

Comment:
We use `=>` syntax for functions, not `function`, and we use `const` instead of `let`.

Reply:
Though honestly, these functions are so small/specific/non-standard to the rest of the formats that we use that there's basically no point to them existing here, just use the `date-fns` functions directly at your specific code.

Reply:
As we need these parts in AdminV3 I separated and added in here

fixed in 7a5572f422c69f780b08342d6dcbaed41d0ed75a

Reply:
We’ll only need them in the utils/csv function that will be used for both the script and admin v3 though. So it would only be in a single location still.

If you’re going to leave the functions in utils/time then they need to be made way more descriptive and robust, named params, talking about the exact kind of date format it’s designed to work for, etc. 

But that doesn’t make sense to do, because they’re not ‘unix philosophy’ ‘single focus’ functions. There are 2 things to do: parse iso, and calculate difference. That’s literally what the date-fns do, therefore there is zero benefit to making these new wrapper functions.

Reply:
fix in 290ef95b0ccb1f291eb9928af210602b6c8bd362